bin/pygrb/pycbc_make_offline_grb_workflow

```diff
@@ -261,7 +281,7 @@ if IPN:
     datafind_veto_files.extend(_workflow.FileList([search_pts_file]))
 
 # Make ExtTrig xml file (needed for lalapps_inspinj and summary pages)
-# TODO: how will this work in O4?
+# TODO: how will this work with the new code
```

Comment:
If you've moved away from lalapps_inspinj, is it needed at all?

Reply:
@spxiwh, this led to a major cleanup.  I removed lalapps_inspinj support from PyGRB.  This knocked off the need to support dark_vs_bright jobs, jitter jobs, align spin jobs from PyCBC in general (the related executables are no longer on master or they are in pylal, which we dropped!).  Finally, all this + `configparser_value_to_file` rendered a couple of functions from `grb_utils.py` obsolete.